Rakefile

```diff
@@ -8,23 +8,23 @@ Bundler::GemHelper.install_tasks
 
 desc "Copy the database.DB.yml per ENV['DB']"
 task :install_database_yml do
-  puts format("installing database.yml for %s", ENV["DB"])
+  puts format("installing database.yml for %s", ENV.fetch("DB", nil))
```

Comment:
These all `ENV.fetch` are not needed and should be reverted.

Reply:
ugh, this came from a bad rebase off of my branch from #1515. will double check and revert